src/app/utils/helpers.js

```diff
@@ -80,3 +80,5 @@ export const formatGas = gasArray => {
 
   return gas
 }
+
+export const truncateString = (string, len) => (string.length >= len ? `${string.slice(0, len - 3)}...` : string)
```

Comment:
What's your opinion on adding in `lodash`? 
There are many functionalities we could leverage, just like the `truncate` function it might come in handy in the future 😄 https://lodash.com/docs/4.17.10#truncate
On the other hand, we do not want to bloat it with packages